getting-started-publishing/login.md

```diff
@@ -0,0 +1,27 @@
+# Tilgang og innlogging på Data.norge.no
+
+For å bruke registreringsløsningen i på Data.norge.no, er det viktig å ha riktig tilgangsnivå og følge korrekt innloggingsprosedyre. Denne veiledningen gir en oversikt over ulike innloggingsmetoder og hvilke tilganger som kreves for å få full utnyttelse av Data.norge.no.
+
+## Innlogging med ID-porten/Altinn
+
+For å få tilgang til registreringsløsningen eller administrasjonsgrensesnittet for høsting på Data.norge.no, må du ha en av følgende rettigheter:
```

Comment:
```suggestion
De aller fleste virksomheter vil logge inn via ID-porten/Altinn for å få tilgang til registreringsløsningen eller administrasjonsgrensesnittet for høsting på Data.norge.no. [På Samarbeidsportalen kan du lese mer om hvordan du får tilgang](https://samarbeid.digdir.no/felles-datakatalog/felles-datakatalog/1617)
```

Reply:
Hvem er unntakene? Hvordan tydeliggjøre for leseren om unntaket gjelder hens virksomhet?